src/rgw/rgw_op.cc

```diff
@@ -2566,6 +2573,9 @@ void RGWListBuckets::execute(optional_yield y)
 
       handle_listing_chunk(std::move(buckets));
     }
+
+    rgw::op_counters::tinc(labeled_counters.get(), l_rgw_op_list_buckets_lat, s->time_elapsed());
```

Comment:
@cbodley is this the most logical place track the latency? Should it be done outside of the while loop?